api/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class NotFoundError(Exception):
+    def __init__(self, error):
```

Comment:
https://stackoverflow.com/questions/1319615/proper-way-to-declare-custom-exceptions-in-modern-python

remove __init__